arbitrum-docs/launch-orbit-chain/orbit-sdk/how-tos/configuring-orbit-chain-with-sdk.md

```diff
@@ -0,0 +1,129 @@
+
+### Setter Functions
+You can use these setter functions to configure the child chain parameters:
+
+- `addChainOwner`: Allows you to add a new chain owner to your Orbit chain.
+
+- `removeChainOwner`: Enables you to remove an existing owner from the list of chain owners.
+
+- `setMinimumL2BaseFee`: Sets the minimum base fee on the child chain. The minimum base fee is the lowest amount that the base fee on the child chain can ever be. For example, the current minimum base fee on Arbitrum One is 0.1 gwei, and on Arbitrum Nova, it is 0.01 gwei.
+
+- `setSpeedLimit`: The fee mechanism on the Arbitrum Nitro stack differs from the Ethereum blockchain. The Nitro stack has a parameter called the speed limit, which targets the number of gas consumed on the child chain per second. If the amount of gas per second exceeds this pre-specified amount, the base fee on the child chain will increase, and vice versa. The current speed limit on Arbitrum One is 7 million gas per second, meaning if the Arbitrum One chain consumes more than 7 million gas per second, its base fee will increase. For more information on the speed limit, please refer to this [document explaining the concept of speed limit in the Nitro stack](https://docs.arbitrum.io/inside-arbitrum-nitro/#the-speed-limit).
+
+- `setInfraFeeAccount`: Sets the infrastructure fee account address, which receives all fees collected on the child chain. It is meant to receive the minimum base fee, with any amount above that going to the network fee account.
+
+- `setNetworkFeeAccount`: Sets the network fee account address. As mentioned, this address collects all fees above the base fee. Note that if you set this amount to the 0 address on your chain, all fees will be deposited into the infrastructure fee account.
+
+- `scheduleArbOSUpgrade`: If you plan to upgrade the <a data-quicklook-from="arbos">ArbOS</a> version of your chain, this method can help you schedule the upgrade. For a complete guide on this matter, please refer to the explanation of the [arbos upgrade](../../how-tos/arbos-upgrade.md).
+
+- `setChainConfig`: We discussed the chainConfig in the [Rollup deployment guide](deploying-rollup-chain-with-sdk.md#chain-config-parameter) in detail. If you wish to change any field of the `chainConfig`, you need to use this method on the child chain.
+
+### Getter Functions
+
+To read the child chain parameters, you can use these getter functions:
+
+- `getAllChainOwners`: Provides the list of all current chain owners.
+
+- `getNetworkFeeAccount`:  Returns the network fee account address.
+
+- `getInfraFeeAccount`:  Returns the infrastructure fee account address.
+
+- `isChainOwner`:  Allows you to check whether an address is on the list of chain owners.
+
+### Configuring the Child Chain Using the Orbit SDK
```

Comment:
```suggestion
### Configure the child chain using the Orbit SDK
```
- nit - sentence-case titles of all kinds ([ref](https://docs.arbitrum.io/for-devs/contribute#style-conventions))
- nit - imperative titles in how-tos (undocumented pattern - [ref](https://www.notion.so/arbitrum/How-to-775951f3d84c476cb6cba296a0a6f671?pvs=4))